templates/webapps/galaxy/visualization/gie.mako

```diff
@@ -0,0 +1,157 @@
+<%inherit file="/webapps/galaxy/base_panels.mako"/>
+
+<%def name="init()">
+<%
+    self.has_left_panel=False
+    self.has_right_panel=False
+    self.active_view="visualization"
+    self.message_box_visible=False
+%>
+</%def>
+
+<%def name="center_panel()">
+
+    ${h.js('libs/jquery/select2')}
+
+    <div style="overflow: auto; height: 100%;">
+        <div class="page-container" style="padding: 10px;">
+            <div class="col-md-8 col-md-offset 2 col-xs-12">
+                <div class="row">
+                    <h1>Galaxy Interactive Environment Launcher</h1>
+                    <p>
+                        Galaxy Interactive Environments (GIEs) launch embedded,
+                        dockerized versions of popular data analysis suites such as
+                        Jupyter and RStudio, right from within Galaxy. They allow you
+                        to dynamically interact with your data, right on the server. No
+                        more uploading and downloading between various platforms just
+                        to get your work done!
+                        <br />
+                        <a href="https://docs.galaxyproject.org/en/master/admin/interactive_environments.html">Admin Docs</a>
+                    </p>
+                    <form id='launcher' action="NONE" method="GET">
+
+                        <table class="table table-striped">
+                            <tr>
+                                <td>GIE: </td>
+                                <td>
+                                    <span id="image_name" style="width: 400px" />
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Image: </td>
+                                <td>
+                                    <span id="image_tag" style="width:400px" />
+                                    <input id="image_tag_hidden" type="hidden" name="image_tag" value="NONE" />
+                                    <p id="image_desc">
+                                    </p>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Datasets: </td>
```

Comment:
Yes, so I've seen. From my perspective with the generic launcher, that's a bit unfortunate. I really like volumes. What's the issue exactly for NFS people?

So, I think we should enable/disable mounting of volumes by an admin user? If disabled, I can just hide `Dataset` selector?